scout/server/blueprints/institutes/controllers.py

```diff
@@ -461,58 +500,25 @@ def cases(store, request, institute_id):
         "vcf_files": 1,
     }
 
-    data["status_ncases"] = store.nr_cases_by_status(institute_id=institute_id)
-    data["nr_cases"] = sum(data["status_ncases"].values())
-
-    sanger_ordered_not_validated: Tuple[Dict[str, list]] = get_sanger_unevaluated(
-        store, institute_id, current_user.email
-    )
-    data["sanger_unevaluated"]: Dict[str, list] = sanger_ordered_not_validated[0]
-    data["sanger_validated_by_others"]: Dict[str, list] = sanger_ordered_not_validated[1]
-
-    # local function to add info to case obj
-    def populate_case_obj(case_obj):
```

Comment:
Moved outside this function now

Reply:
Ok, I very much approve of the general change, but only proof-read this cursorily.